modules/features/falcon_feature_user/config/install/user.role.appeals_manager.yml

```diff
@@ -12,15 +12,19 @@ permissions:
   - 'access toolbar'
   - 'create appeal content'
   - 'create media'
+  - 'create source_code entities'
```

Comment:
Why would source code permissions go into falcon_feature_user? Correct me if I'm wrong but I thought it should go to falcon_feature_thankq.

Reply:
It does trigger me too... only if there is some issue with having permissions that are not defined (due to some modules not installed as chosen by client project) inside the falcon_feature_user - only in this case I would go with the roles permissions overriding in this feature... and still it means we would need to update this config here every time we change permissions (along with falcon_feature_user configs) - doesn't look maintainable at all...

Reply:
@kalabro it's a bug, thanks, fixed.